src/config/theme.js

```diff
@@ -24,34 +24,46 @@ const lineHeightLg = '133.33%';
 const lineHeightSm = '125%';
 const lineHeight32 = '32px';
 
-// Neon blue
-const primaryColor = '#585AFA';
-const primaryColorDark = '#4244BB';
-const primaryColorLight = '#F0F0FF';
+// Brand Colors
+const primaryBrandColor = '#585afa';
+const secondaryBrandColor = '#23dae0';
 
-// Neo teal
-const secondaryColor = '#23DAE0';
-const secondaryColorLight = '#E9FEFE';
-const secondaryColorDark = '#11A5A9';
+// Brand Color variables
+const primaryBrandColor200 = '#4648c8';
+const primaryBrandColor80 = '#797bfb';
+const primaryBrandColor60 = '#9b9cfc';
+const primaryBrandColor30 = '#cdcefe';
+const secondaryBrandColor200 = '#1caeb3';
+const secondaryBrandColor80 = '#4fe1e6';
+const secondaryBrandColor60 = '#7be9ec';
+const secondaryBrandColor30 = '#bdf4f6';
 
-// White
-const white = '#FFFFFF';
+// Text Colors
+const textColorDarkGrey = '#333333';
+const textColorMediumGrey = '#666666';
+const textColorLightGrey = '#b3b3b3';
 
-// Red
-const redColor = '#FE4A49';
-const redColorLight = '#FFDEDE';
-const redColorDark = '#960F0E';
+// Background Color
+const backgroundColorLight = '#f9f9f9';
```

Comment:
Will we have other darker background colors?

Reply:
Sorry, I don't know. But now there is only one background color.
![image](https://user-images.githubusercontent.com/18548885/47837329-995c9c00-dde6-11e8-9d24-d4b509da4fd8.png)